src/BgZ-3-0/Cert/_reference/resources/medmij-bgz-encounter-ts-01.xml

```diff
@@ -21,32 +21,26 @@
       <individual>
          <extension url="http://nictiz.nl/fhir/StructureDefinition/practitionerrole-reference">
             <valueReference>
-               <reference value="PractitionerRole/medmij-bgz-practitionerrole-ts-03"/>
-               <display value="Orthopedie"/>
+               <reference value="PractitionerRole/medmij-bgz-practitionerrole-ts-05"/>
+               <display value="SEH-arts"/>
             </valueReference>
          </extension>
-         <reference value="Practitioner/medmij-bgz-practitioner-ts-01"/>
-         <display value="G.Z.M. de Wit"/>
+         <reference value="Practitioner/medmij-bgz-practitioner-ts-04"/>
+         <display value="J.H.R. Peters"/>
       </individual>
    </participant>
    <period>
-      <start value="2012-08-16"/>
-      <end value="2012-08-19"/>
+      <start value="2012-08-15"/>
+      <end value="2012-08-15"/>
    </period>
-   <diagnosis>
-      <condition>
-         <reference value="Procedure/medmij-bgz-procedure-ts-02"/>
-         <display value="Surgical reduction of fracture (procedure)"/>
-      </condition>
-   </diagnosis>
    <diagnosis>
       <condition>
          <reference value="Condition/medmij-bgz-condition-ts-05"/>
          <display value="Fracture of lower limb (disorder)"/>
```

Comment:
Kunnen we deze display-waarde ook meteen in het Nederlands neerzetten (net als in Condition-resource zelf)?